Document/Document_Files/cm1.html

```diff
@@ -697,5 +732,5 @@ <h2>Class Statistics : </h2>
 </tr>
 </table>
 </body>
-<p style="text-align:center;border-top:1px solid black;">Generated By <a href="http://www.pycm.ir" style="text-decoration:none;color:red;">PyCM</a> Version 2.5</p>
+<p style="text-align:center;border-top:1px solid black;">Generated By <a href="http://www.pycm.ir" style="text-decoration:none;color:red;">PyCM</a> Version 2.6</p>
 </html>
```

Comment:
some files have been left without last newline in this commit, if you think it is necessary you can add newline to them or i can do this for you :smile:.

Reply:
I think it's not very important in `.html` file ;-)